components/authentication-framework/org.wso2.carbon.identity.application.authentication.endpoint.util/src/main/java/org/wso2/carbon/identity/application/authentication/endpoint/util/Constants.java

```diff
@@ -120,4 +120,19 @@ private UserRegistrationConstants() {
 
         }
     }
+
+    public static class ErrorToi18nMappingConstants {
+
+        public static final String INVALID_CALLBACK_CALLBACK_NOT_MATCH = "invalid_callback_callback.not.match";
+        public static final String INVALID_CALLBACK_CALLBACK_NOT_MATCH_I18N_KEY = "callback.not.match";
+        public static final String INVALID_CLIENT_APP_NOT_FOUND = "invalid_client_application.not.found";
+        public static final String INVALID_CLIENT_APP_NOT_FOUND_I18N_KEY = "application.not.found";
+        public static final String INVALID_REQUEST_INVALID_REDIRECT_URI = "invalid_request_invalid.redirect.uri";
```

Comment:
Can we use `invalid_request.redirect.uri`, instead of using `invalid` twice.

Reply:
This is a combination of 2 error codes. Hence we cannot remove the repetition here.